src/styles/utiles/_normalize.scss

```diff
@@ -0,0 +1,8 @@
+html {
+  font-family: "Arial", sans-serif;
+}
+
+.body {
```

Comment:
You can delete this .body class with display and justify-content also in your html too, and center your .calendar with margin: 0 auto, it will be more compact
```suggestion

```

Reply:
> You can delete this .body class with display and justify-content also in your html too, and center your .calendar with margin: 0 auto, it will be more compact

I use it only as a container in this task, because i can`t add margin to BEM block .calendar